.travis.yml

```diff
@@ -2,14 +2,12 @@ language: java
 
 sudo: false
 
-script:
-  - jdk_switcher use oraclejdk7
-  - mvn clean verify -P jdk7
-  - jdk_switcher use oraclejdk8
-  - mvn clean verify -P jdk8
+jdk:
+  - oraclejdk8
+  - oraclejdk7
 
-after_success:
-  - mvn clean test jacoco:report coveralls:report
+script:
```

Comment:
> We should probably change all our projects to do this.

Hm, not sure I intended to do this, missing the coverage metrics would now fail the build. Using `after_success` we'd get a successful build with a separate report on the pr regarding coverage.